recipes-core/packagegroups/packagegroup-ni-coreimagerepo.bb

```diff
@@ -34,6 +34,7 @@ RDEPENDS_${PN}_append_x64 = "\
 	nilrt-grub-runmode \
 	nilrt-grub-safemode \
 	${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'packagegroup-ni-xfce', '', d)} \
+	onboard \
```

Comment:
I know the naming isn't ideal, but could you put this dependency into [`packagegroup-ni-xfce`](https://github.com/ni/meta-nilrt/blob/nilrt/master/hardknott/recipes-core/packagegroups/packagegroup-ni-xfce.bb#L12) with the other graphical applications, rather than loose in the coreimagrepo.

As a part of my more ambitious packagegroup refactors in meta-nilrt #300, I'm trying to get all the loose packages out of what will be `packagefeed-ni-core`.